runtime/gc_base/ScavengerForwardedHeader.hpp

```diff
@@ -23,17 +23,6 @@
 #if !defined(SCAVENGERFORWARDEDHEADER_HPP_)
 #define SCAVENGERFORWARDEDHEADER_HPP_
 
-#include "j9.h"
-#include "j9cfg.h"
```

Comment:
This should not assume anything is include before this file, which needs things like `uintptr_t` and `OMR_GC_COMPRESSED_POINTERS`.